coral/defaults.env

```diff
@@ -1,9 +1,13 @@
-CHAIN_ID="cosmwasm-coral"
-TESTNET_NAME="coralnet"
-WASMD_VERSION="v0.10.0"
-COSMJS_VERSION="v0.22.0"
+export CHAIN_ID="cosmwasm-coral"
+export TESTNET_NAME="coralnet"
+export WASMD_VERSION="v0.10.0"
+export CONFIG_DIR=".corald"
+export BINARY="corald"
 
-RPC="https://rpc.coralnet.cosmwasm.com"
-LCD="https://lcd.coralnet.cosmwasm.com"
-FAUCET="https://faucet.coralnet.cosmwasm.com"
-SEED_NODE="26c9c79dc62b5ddc753bb9fcce022fcc98b5a8cf@p2p.demo-09.cosmwasm.com:26656"
+export COSMJS_VERSION="v0.22.0"
+export GENESIS_URL=https://github.com/CosmWasm/testnets/raw/master/coral/config/genesis.json
```

Comment:
This gives a redirect. Please use: https://raw.githubusercontent.com/CosmWasm/testnets/master/coral/config/genesis.json